BUILD.bazel

```diff
@@ -44,6 +44,11 @@ filegroup(
         "//examples:all-srcs",
         "//federation:all-srcs",
         "//hack:all-srcs",
+        "//kubernetes/cluster:all-srcs",
```

Comment:
Turns out I had a kubernetes/ directory under KUBE_ROOT, possibly as build output. Removing that did the trick. Thanks!